app/components/works/dates_component.rb

```diff
@@ -16,8 +16,8 @@ def date_range_start_year
       number_field_tag "#{prefix}[created_range(1i)]", created_range_start_year,
                        data: {
                          date_validation_target: 'year',
-                         date_range_target: 'year',
-                         action: 'date-validation#change date-range#change'
+                         date_range_target: 'startYear',
+                         action: 'date-range#change clear->date-validation#clear validate->date-validation#validate'
```

Comment:
Notice the binding of the Date Validation controller to the events that will be emitted by the Date Range controller.